src/quality-patches/release-notes.md

```diff
@@ -16,34 +16,16 @@ The [Magento Quality Patches](https://github.com/magento/quality-patches) packag
--  MDVA-30052: Customer data section invalidation logic has been improved. This release introduces a new way of invalidating all customer sections data that avoids a known issue with local storage when custom `sections.xml` invalidations are active. (Previously, private content (local storage) was not correctly populated when you had a custom *etc/frontend/sections.xml* with action invalidations). See https://devdocs.magento.com/guides/v2.3/extension-dev-guide/cache/page-caching/private-content.html#invalidate-private-content
--  :
--  MDVA-30131: Issue fixed: "No" values are not included in layered nav aggregation options when using ElasticSearch as the search engine. This affected storefront and GraphQl scenarios in the same way.
--  :
--  Root cause: There were issues due to the way PHP handles false values.":
-
-## v1.0.2 example 2
-
-{:.bs-callout-warning}
-**Editorial note:** The following release notes sample (incomplete) manually structured and entered from the 1.0.2 [`patches.json`](https://github.com/magento/quality-patches/pull/28/files#diff-2946f7f291517d21cc254a82219fdb51) file in the package repo.
-
--  **Patches for Magento 2.3.0 - 2.4.0**
-   -  **MDVA-26694**—Product and catalog caches will expire as scheduled. Previously, caches expire daily, as сron runs the `catalogrule_apply_all` task once a day which re-indexes all the catalog rules and its dependent indexers and clears the cache for all products and categories.
-   -  **MDVA-27825**—Big data customer export issue. Memory leak prevented by adding clearing of Customer objects in `\\Magento\\Framework\\Model\\ResourceModel\\Db\\VersionControl\\Snapshot` class.
-
--  **Patches for Magento 2.3.2-p2 - 2.3.5**
-   -  **MDVA-30052**—Customer data section invalidation logic has been improved. This release introduces a new way of invalidating all customer sections data that avoids a known issue with local storage when custom `sections.xml` invalidations are active. (Previously, private content (local storage) was not correctly populated when you had a custom `etc/frontend/sections.xml` with action invalidations).
+See [Apply patches]({{ site.baseurl }}/guides/v2.4/comp-mgr/patching.html) for instructions on applying patches to your Magento projects.
+
+## v1.0.2
+
+-  **MDVA-25602** _(for Magento `>=2.3.0 <2.3.2`, `>=2.3.0 <2.3.5`, `>=2.3.2 <2.3.3`, `>=2.3.3 <2.3.5`)_—Fixes issue with PayPal Payflow Pro payment method and treating cookies as SameSite=Lax by default in the Chrome 80 browser and API response redirect to customer login page.
```

Comment:
I suggest replacing with a more simple format for customers: 2.3.0 - 2.3.4. It includes all versions above